Scripts/gen_Primary.sh

```diff
@@ -45,7 +45,7 @@ usage() {
   [ --run (opt) default 1202 ]
   [ --cat(opt) default Cat ]
 
-  bash gen_Primary.sh --primary DIOTail --type MuMinus --campaign MDC2020 -pver z_sm3 --sver p --njobs 100 --events 100 --start 75 --end 95
+  bash gen_Primary.sh --primary DIOTail --type MuMinus --campaign MDC2020 -pver MDC2020z_sm3 --sver MDC2020p --njobs 100 --events 100 --start 75 --end 95
```

Comment:
I believe the previous version is correct, at least at the head of Production.  Maybe you need to merge the head of main into your branch to test.
https://github.com/Mu2e/Production/blob/40aa536a7ffa822bd6a4027ce5995e21fd5b73ed/Scripts/gen_Primary.sh#L120C2-L120C36